teku/src/main/java/tech/pegasys/teku/cli/options/ExecutionEngineOptions.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.cli.options;
+
+import static tech.pegasys.teku.config.TekuConfiguration.Builder;
+
+import java.util.ArrayList;
+import java.util.List;
+import picocli.CommandLine.Option;
+
+public class ExecutionEngineOptions {
+
+  @Option(
+      names = {"--ee-endpoints", "--ee-endpoint"},
```

Comment:
I suspect these should have `--X` prefix so they appear in the `teku -X` output. Otherwise they'll be impossible to discover.